src/source.rs

```diff
@@ -112,6 +112,21 @@ pub enum ResolveResponse {
   Umd(ModuleSpecifier),
 }
 
+impl ResolveResponse {
+  pub fn to_result(self) -> Result<ModuleSpecifier, Error> {
```

Comment:
Maybe `to_specifier` instead?

Reply:
Hmmm... I would prefer result, because it unwraps the enum to a result.

Reply:
What about `try_specifier()` or something like that?